core/editor/templates/editor-body-v2.view.php

```diff
@@ -0,0 +1,12 @@
+<?php
+namespace Elementor\Core\Editor\Templates;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+?>
+
+<div id="elementor-editor-wrapper-v2"></div>
```

Comment:
```suggestion
<div id="e-editor-v2-wrapper"></div>
```
maybe?

Reply:
This is a copy of this:
<img width="313" alt="image" src="https://user-images.githubusercontent.com/20494189/206201229-fc5e56a5-37f9-4f61-b874-3f8de4fafbad.png">